python/hail/representation/genomeref.py

```diff
@@ -49,7 +49,6 @@ def __init__(self, name, contigs, lengths, x_contigs=[], y_contigs=[], mt_contig
         mt_contigs = wrap_to_list(mt_contigs)
 
         par_strings = ["{}:{}-{}".format(contig, start, end) for (contig, start, end) in par]
-        self._par_jrep = Env.hail().variant.Locus.parseIntervals(wrap_to_list(par_strings))
```

Comment:
For my clarity, changes to `genomeref.py` don't have anything to do with implicit ordering, right? It looks like a conversion to lazy loading of many attributes (which looks independently great).

Reply:
Sorry -- this was in a stacked PR Tim reviewed already #2534. I should figure out how to stack them properly like you did.